apps/docs/src/app/structure.ts

```diff
@@ -659,6 +660,18 @@ const structure: DocsStructure = makeStructure({
                     hasExamples: false,
                     isNew: expiresAt('2025-09-20')
                 },
+                {
+                    id: DocsStructureItemId.NotificationCenter,
+                    name: {
+                        ru: 'Notification Center',
+                        en: 'Notification Center'
+                    },
+                    svgPreview: 'notification-center',
+                    hasApi: true,
+                    apiId: 'notification-center',
+                    hasExamples: false,
+                    isNew: expiresAt('2025-10-30')
```

Comment:
Пусть до конца октября повисит 🙂 
```suggestion
                    isNew: expiresAt('2025-10-31')
```

Reply:
Добавил еще больше... пока вольем время пройдет